.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: trusty
 addons:
   apt:
     sources:
-      - chef-stable-trusty
+      - chef-current-trusty
```

Comment:
People tend to install stable on their workstations and can have CI fail because current has a new version of something.  It's possible to use appbundle-updater in such cases but probably something we want to avoid.

I'd prefer to keep CI version the same as latest workstation version.

Thoughts?  cc @damacus